tests-beta/ios/MASVS-PLATFORM/MASTG-TEST-0279.md

```diff
@@ -5,7 +5,22 @@ id: MASTG-TEST-0279
 type: [dynamic]
 weakness: MASWE-0053
 threat: [app]
-status: placeholder
-note: This test checks if the app sets an expiration date for the contents of the general pasteboard using the `UIPasteboard.setItems(_:options:)` method with the `UIPasteboard.Options.expirationDate` option.
 profiles: [L2]
 ---
+
+## Overview
+
+This test checks if the app sets an expiration date for the contents of the general [pasteboard](../../../Document/0x06h-Testing-Platform-Interaction.md/#pasteboard) using the `UIPasteboard.setItems(_:options:)` method with the `UIPasteboard.Options.expirationDate` option. If sensitive data is left in the pasteboard without an expiration date, it can be accessed by other apps indefinitely, leading to potential data leaks.
+
+## Steps
+
+1. Run a static analysis scan using @MASTG-TOOL-0073 to detect usage of the [`UIPasteboard.general`](https://developer.apple.com/documentation/uikit/uipasteboard/1622106-generalpasteboard "UIPasteboard generalPasteboard") property.
```

Comment:
Shouldn't tests have TECH references rather than TOOL references? 0073 seems to fit, but should be expanded a bit to show usages with tools

Reply:
Once I have finished another PR that does exactly this, I will refactor all tests to use TECH. This enhances 0073 and many other tests related to reverse engineering. For now, I'll leave these as they are.